lib/src/parse/sass.dart

```diff
@@ -48,12 +48,23 @@ class SassParser extends StylesheetParser {
     do {
       buffer.addInterpolation(almostAnyValue());
       buffer.writeCharCode($lf);
-    } while (buffer.trailingString.trimRight().endsWith(",") &&
-        scanCharIf(isNewline));
+    } while (_endsWithComma(buffer.trailingString) && scanCharIf(isNewline));
 
     return buffer.interpolation(scanner.spanFrom(start));
   }
 
+  // Returns true if [text] ends with a comma, ignoring whitespace and comments.
+  bool _endsWithComma(String text) {
```

Comment:
I'm concerned that this doesn't respect CSS tokenization rules. For example, if `"//"` somehow made it into the selector, it would be considered a comment here rather than a string. Since `almostAnyValue()` does fully tokenize, maybe we can get it to be a little more clever here. Maybe add an `omitComments` parameter that tells it not to include comments in the buffer?

Reply:
Yeah, that makes sense. Originally, I didn't think that would work, since the comments would be omitted in the output, but since even loud comments are omitted within selectors, it should be fine.

Reply:
Done